releasenotes/notes/pulse-gate-calibrations-78fd3fa5a5328761.yaml

```diff
@@ -0,0 +1,36 @@
+---
+features:
+  - |
+    Circuits now support a new feature: Pulse gates. This feature enables users who are working
+    primarily with ``QuantumCircuits`` to override (for basis gates) or specify (for standard
+    and custom gates) a definition of a ``Gate`` operation in terms of time-ordered signals
+    across hardware channels. In other words, it allow users to provide a pulse-level "custom
+    gate calibration."
+
+    The circuits are built exactly as before. For example::
+
+        from qiskit import pulse
+        from qiskit.circuit import QuantumCircuit, Gate
+
+        class RxGate(Gate):
+            def __init__(self, theta):
+                super().__init__('rxtheta', 1, [theta])
```

Comment:
The naming/choice of gate is a little confusing when there's a pre-existing Qiskit ``RXGate``.